nirodents/workflows/brainextraction.py

```diff
@@ -156,14 +146,11 @@ def init_rodent_brain_extraction_wf(
     wf = pe.Workflow(name)
 
     # truncate target intensity for N4 correction
-    clip_target = pe.Node(niu.Function(function=_trunc), name="clip_target")
-    clip_target.inputs.percentiles = (None, 99.9)
-    clip_target.inputs.clip_max = None
+    clip_target = pe.Node(IntensityClip(p_min=0, p_max=99.9), name="clip_target")
```

Comment:
```suggestion
    clip_target = pe.Node(IntensityClip(p_min=15.0, p_max=95.0), name="clip_target")
```

Reply:
perhaps more conservatively:

```suggestion
    clip_target = pe.Node(IntensityClip(p_min=5, p_max=98), name="clip_target")
```

(like the template)

Reply:
After doing some tests locally, the above suggestions weren't quite right (at least not for the test data that runs on circleCI. These (compromise) settings worked well for the target: 
```suggestion
    clip_target = pe.Node(IntensityClip(p_min=15, p_max=99.9), name="clip_target")
```

Reply:
I suppose we'll only find out if it's overfitting to one image once we test on more than one image...